src/filters/FilterSystem.ts

```diff
@@ -225,7 +225,23 @@ export class FilterSystem implements System
 
         this._calculateFilterArea(instruction, bounds);
 
-        this._calculateFilterBounds(filterData, renderer.renderTarget.rootViewPort, rootAntialias, rootResolution, 1);
+        let viewport = renderer.renderTarget.rootViewPort;
+
+        const renderGroup = instruction.container.renderGroup || instruction.container.parentRenderGroup;
```

Comment:
`instruction.container` is not always available. This line will break execution and produce "TypeError".
`FilterInstruction` contains either container or renderables.